conceptnet5/readers/cc_cedict.py

```diff
@@ -0,0 +1,252 @@
+"""
+CC-CEDICT is a continuation of the CEDICT project started by Paul Denisowski in 1997 with the aim
+to provide a complete downloadable Chinese to English dictionary with pronunciation in pinyin
+for the Chinese characters.
+
+Creative Commons Attribution-Share Alike 3.0
+http://creativecommons.org/licenses/by-sa/3.0/
+
+Referenced works:
+CEDICT - Copyright (C) 1997, 1998 Paul Andrew Denisowski
+
+CC-CEDICT can be downloaded from:
+http://www.mdbg.net/chindict/chindict.php?page=cc-cedict
+"""
+
+import gzip
+import re
+
+from conceptnet5.edges import make_edge
+from conceptnet5.formats.msgpack_stream import MsgpackStreamWriter
+from conceptnet5.nodes import standardized_concept_uri
+from conceptnet5.uri import Licenses
+
+DATASET = '/d/cc_cedict'
+LICENSE = Licenses.cc_sharealike
+SOURCE = [{'contributor': '/s/resource/cc_cedict/2017-10'}]
+
+LINE_REGEX = re.compile(r'(.+)\s(.+)\[.+\]\s/(.+)/')  # separate traditional and simplified words
+DATE_RANGE_REGEX = re.compile(r'(.+?)\s\(.+\d.+\),')  # date range
+PAREN_REGEX = re.compile(r'\(.+?\)')  # parenthesis
+CHINESE_CHAR_REGEX = re.compile(r'([\u4e00-\u9fff]+[\|·]?)+')  # Chinese characters
+BRACKETS_REGEX = re.compile(r'\[.+\]')  # pronunciation
```

Comment:
I think this regex is too greedy and should have a `?` on it like `PAREN_REGEX` does.

I believe that in this definition:

    一甲 一甲 [yi1 jia3] /1st rank or top three candidates who passed the imperial examination (i.e. 狀元|状元[zhuang4 yuan2], 榜眼[bang3 yan3], and 探花[tan4 hua1], respectively)/

it will match this text:

    [zhuang4 yuan2], 榜眼[bang3 yan3], and 探花[tan4 hua1]

Reply:
This particular definition would actually have everything inside the parentheses removed before matching the brackets, but the problem was true for a couple of other definitions, so I changed it.